appserver/tests/payara-samples/pom.xml

```diff
@@ -103,29 +118,11 @@
             <dependency>
                 <groupId>io.opentracing</groupId>
                 <artifactId>opentracing-api</artifactId>
-                <version>0.30.0</version>
+                <version>0.32.0</version>
```

Comment:
Not sure. What do you think, @pdudits ?

Reply:
Btw - BOM maybe should use payara.version property instead of project.version ... but then samples cannot be executed with older version. There will be always issues with compatibility if we would try to run samples with another version than the current.
I would prefer using jakartaee-bom here and not payara.bom, because we use directly the zip file, payara-bom should not be needed here.

Reply:
The answer is "yes" to both points ;)

Opentracing API is included in the BOM

https://github.com/payara/Payara/blob/master/api/payara-bom/pom.xml#L586

Samples should use payara.version to import the BOM. Because it is not just about Jakarta APIs, it is also about microprofile APIs, opentracing apis, and jersey implementations for client-side tests (which break in embedded mode if they differ).

Reply:
On the other hand, the reason for moving samples into the main tree was, that you should be running samples against same matching payara version. But adding indirection with `payara.version` allows for some wiggle room between snapshot and release versions.